GitHubDeployments.ps1

```diff
@@ -0,0 +1,567 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+@{
+    GitHubDeploymentEnvironmentTypeName = 'GitHub.DeploymentEnvironment'
+}.GetEnumerator() | ForEach-Object {
+    Set-Variable -Scope Script -Option ReadOnly -Name $_.Key -Value $_.Value
+}
+
+filter New-GitHubDeploymentEnvironment
+{
+    <#
+    .SYNOPSIS
+        Creates a new deployment environment on a GitHub repository.
+
+    .DESCRIPTION
+        Creates a new deployment environment on a GitHub repository.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Uri
+        Uri for the repository.
+        The OwnerName and RepositoryName will be extracted from here instead of needing to provide
+        them individually.
+
+    .PARAMETER EnvironmentName
+        The name of the environment.
+
+    .PARAMETER WaitTimer
+        The amount of time to delay a job after the job is initially triggered.
+        The time (in minutes) must be an integer between 0 and 43,200 (30 days).
+
+    .PARAMETER DeploymentBranchPolicy
+        Whether only branches with branch protection rules or that match the specified name patterns
+        can deploy to this environment.
+
+    .PARAMETER ReviewerTeamId
+        The teams that may review jobs that reference the environment.
```

Comment:
From implementation, it looks like this is isn't an "either or" situation (which is how this comment makes it out to be).  It might be more clear if written as:

`You can provide a combination of up to six users and/or teams as reviewers.`.

I understand that you were merely matching what the API docs say, but I think they are confusing as well.

Here and elsewhere (assuming you've actually tested that you _can_ provide a combination of both teams and users as reviewers since there's no UT that validates that).